vault/dendron.topic.publish.cook.github.md

```diff
@@ -22,6 +22,8 @@ Before continuing: **we do not recommend this path for publishing.** If possible
 
 - You have [[Dendron CLI|dendron.ref.cli]] installed (version `0.84.0` or greater)
 
+![[Local directory|dendron://dendron.dendron-site/dendron.topic.cli#installation:#common-options]]
```

Comment:
one nitpick - you want to transclude just the local installation because that's required for github actions. so `![[dendron.topic.cli#local-directory,1:#*]]`

Reply:
Got it. Should I amend the following to highlight that?

> You have [[Dendron CLI|dendron.ref.cli]] installed *locally* (version `0.84.0` or greater)

Reply:
yep, that and updating the note reference and we should be good to go